init/config.js

```diff
@@ -0,0 +1,31 @@
+module.exports = {
+  google: {
+    enabled: true,
+    key: "27004093375-056i2bnbk9g1bpuhbe642asvcl9da5p5.apps.googleusercontent.com",
```

Comment:
Where is this from? Let's not commit secrets into the repo.

Reply:
We have to do something about secrets, me and Wolfgang had a discussion about having this here and it seems like a scary idea but at the same time we have no other option as the auth flow depends on google.

We can delegate creating these secrets to the developer but this will create a bit of friction and I doubt front end devs would undergo the process of creating auth layer on their own google dashboard.

Reply:
Can we get this from an env var, and we give it to developers during onboarding?

Reply:
Sorry, missed I need to submit my comment, and it was pending for two days. 
As I mentioned, ideally we want to have minimum friction during onboarding and the easiest approach is to use separate Google Auth Client App for local testing. Thus, I created new project in my google admin console to make sure we are not exposing prod/staging creds.